quick-github-guide.md

```diff
@@ -0,0 +1,163 @@
+# Quick guide to working with a GitHub repo
+
+This page is intended for people who want to update the
+[Kubeflow docs](https://www.kubeflow.org/docs/) and
+who don't use git or GitHub often. The page gives you a quick guide to
```

Comment:
git -> Git

I interpret this as using Git the product rather than git the command.

Reply:
Done.